portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/IFrame.path

```diff
@@ -62,7 +62,7 @@
 </tr>
 <tr>
 	<td>DIALOG_BOX</td>
-	<td>//div[contains(@class,'modal-header')]/h3</td>
+	<td>//div[contains(@class,'modal-title')]</td>
```

Comment:
@davisaints do you think we can use some existing path/s in Modal.path so we don't create paths that could be duplicates

Reply:
This change is no longer necessary as it has already been made in another pr.